src/harness/cbsd_sas_harnes/cbsd_to_sas.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python
+# -- encoding: utf-8 --
+
+# Project: CBSD to SAS interface
+__author__ = """
+    Kaushik Bekkem (kaushik.bekkem@federatedwireless.com),
+    Francisco Benavides (francisco.benavides@federatedwireless.com) """
+"""
+    CBSD to SAS Interface
+    Federated Wireless
+
+    SAS to User Protocol Technical Specification
+    WINNF-16-S-0016 SAS to CBSD v0.3.10, 12 July 2016
+
+    9.2     Message Transport
+            HTTP v1.1, Content-type: application/json
+            JSON objects shall use UTF-8 enconding
+            All requests from CBSD to SAS shall use the HTTP POST method
+"""
+
+# Imports
+import logging
+import sys
+import os
+from datetime import datetime
+from pytz import timezone
+# Import OWN
+from sas_interface import SAS_Interface
+from sas_interface import PUSH
+
+# Logging file path
+LOG_FILE = 'log/cbsd-to-sas-harness.log'
+# Log format
+LOG_FORMAT = logging.Formatter(
+    '%(asctime)-15s - CBSD to SAS Harness - '
+    '%(funcName)s - %(levelname)s - %(message)s')
+# CBSD to SAS configuration file sections and options
+#INI_FILE_SECTION = 'CBSD-SAS'
+#INI_FILE_OPTIONS = {
+#    'CAT_A': None,
+#    'CAT_B': None,
+#    'SPECTRUM_GAA': None,
+#    'SPECTRUM_PAL': None,
+#    'GRANT': None,
+#    'HEARTBEAT': None}
+# CBSD to SAS Resources
+RESOURCES = {
+    'reg': {
+        'name': 'REGISTRATION',
+        'path': '/registration',
+        'req': 'registrationRequest',
+        'res': 'registrationResponse',
+        'ini': 'REGISTRATION'},
+    'dereg': {
+        'name': 'DEREGISTRATION',
+        'path': '/deregistration',
+        'req': 'deregistrationRequest',
+        'res': 'deregistrationResponse',
+        'ini': 'DEREGISTRATION'},
+    'spec': {
+        'name': 'SPECTRUM INQUIRY',
+        'path': '/spectrumInquiry',
+        'req': 'spectrumInquiryRequest',
+        'res': 'spectrumInquiryResponse',
+        'ini': 'SPECTRUM'},
+    'grant': {
+        'name': 'GRANT',
+        'path': '/grant',
+        'req': 'grantRequest',
+        'res': 'grantResponse',
+        'ini': 'GRANT'},
+    'hb': {
+        'name': 'HEARTBEAT',
+        'path': '/heartbeat',
+        'req': 'heartbeatRequest',
+        'res': 'heartbeatResponse',
+        'ini': 'HEARTBEAT'},
+    'rel': {
+        'name': 'RELINQUISHMENT',
+        'path': '/relinquishment',
+        'req': 'relinquishmentRequest',
+        'res': 'relinquishmentResponse',
+        'ini': 'RELINQUISHMENT'}}
+# Response Code values
+SUCCESS = 0
```

Comment:
1. Please look at new branch 'patch-3' where I have made the 'merge' in terms of not affecting our already running test scripts
2. Removed code never used

Ok on the constant values as mentioned